pages.pt_BR/common/alex.md

```diff
@@ -0,0 +1,21 @@
+# alex
+
+> Uma ferramenta que captura escrita insensível e sem consideraçāo.
+> Ajuda a encontrar no texto, frases favorecedoras de gênero, polarizantes, relacionadas à raça, insensíveis à religiao e outras frases desiguais.
+> Mais informações: <https://github.com/get-alex/alex>.
+
+- Analisa o texto do stdin:
+
+`echo {{His network looks good}} | alex --stdin`
```

Comment:
Please translate the placeholders for get this easier for non-English speakers.

Reply:
Done :)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good